translations/messages.fr.xlf

```diff
@@ -0,0 +1,3061 @@
+                    &lt;li&gt;Pick the capture(s) resulting from your move (by clicking on it).&lt;/li&gt;
+                &lt;/ol&gt;
+            &lt;/li&gt;
+        &lt;/ol&gt;&lt;br/&gt;
+        You&apos;re playing Dark, you can make a capture, do it.</source>
+        <target>Pour faire une capture, il faut aligner 4 de ses propres pièces, qui seront les 4 premières capturées.
+        Il y a plusieurs choses à savoir sur une capture :
+        &lt;ol&gt;
+            &lt;li&gt;Quand 4 pièces sont capturées, toutes les pièces directement alignées avec ces 4 pièces le sont également.&lt;/li&gt;
+            &lt;li&gt;Dès qu&apos;il y a une case vide dans la ligne, la capture s&apos;arrête.&lt;/li&gt;
+            &lt;li&gt;Vos pièces capturées rejoignent votre réserve.
+                Celles de l&apos;adversaire par contre sont réellement capturées et ne rejoignent pas sa réserve.&lt;/li&gt;
+            &lt;li&gt;Si vous créez une ligne de 4 pièces de l&apos;adversaire, c&apos;est au début de son tour qu&apos;il pourra les capturer.
+                Ceci implique que votre tour se passe en trois phases :
+                &lt;ol type=&quot;A&quot;&gt;
+                    &lt;li&gt;Choisir la/les capture(s) crée(s) par le dernier mouvement de votre adversaire.&lt;/li&gt;
+                    &lt;li&gt;Faire votre poussée.&lt;/li&gt;
+                    &lt;li&gt;Choisir la/les ligne(s) à capturer que vous venez de créer (en cliquant dessus).&lt;/li&gt;
+                &lt;/ol&gt;
+            &lt;/li&gt;
+        &lt;/ol&gt;&lt;br/&gt;
+        Vous jouez Foncé, une capture est faisable, faites la.</target>
+      </trans-unit>
+      <trans-unit id="8768850104658663274" datatype="html">
+        <source>Congratulations, you have gotten 4 of your pieces back. This is not the most useful capture.
+        Let&apos;s now see how captures can really be useful.</source>
+        <target>Bravo, vous avez récupéré 4 de vos pièces, mais ce n&apos;est pas la capture la plus utile.
+        Voyons maintenant la vraie utilité d&apos;une capture.</target>
+      </trans-unit>
+      <trans-unit id="5296311699765609481" datatype="html">
+        <source>Capturing (2/3)</source>
+        <target>Capture (2/3)</target>
+      </trans-unit>
+      <trans-unit id="6012182624396746560" datatype="html">
+        <source>Here, there are three different ways of capturing.
+        One way does not capture any of your opponent&apos;s pieces.
+        Another way captures one of your opponent&apos;s pieces.&lt;br/&gt;&lt;br/&gt;
+        The last way captures two of your opponent&apos;s pieces, pick that one.</source>
+        <target>Ici, il est possible de capturer de trois façons différentes.
+        L&apos;une ne permet aucune capture de pièce adverse.
+        L&apos;autre permet une capture de pièce adverse.&lt;br/&gt;&lt;br/&gt;
+        La dernière en permet deux, choisissez cette dernière.</target>
```

Comment:
Si on faisait plutôt 
Ici blblbl
1
2
3
double br
Faite le 3
 ?

Reply:
Aussi, en anglais, au pluriel quand il n'y en a aucunes ?(no opponent's pieceS" ?

Reply:
Yep! En anglais, si c'est comptable, not any est pluriel oui ! (Alors qu'en français, "aucunes" est quasi systématiquement une faute)